### DISCUSSION TITLE
Rank Mirrors Addon Bug

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#379] [Fixed Issues]
Addon gives an error message and puts the .au mirror at the top of the list, even though it ranks badly.

**Comment by Sagnasty** [comment]
The behavior I have been seeing is that it posts the results from the ZA mirror and then hangs.

![mirrors](https://user-images.githubusercontent.com/76575081/122257708-42743a00-ce9e-11eb-8043-befd2f17a75d.jpg)

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by rern** [comment]
@R101-Github,
Jim,

The addon has been improved. Please try again.

**Reply by R101G** [reply]
Works well for me. Thanks rern.
